static/styles/image_upload_widget.css

```diff
@@ -161,16 +183,28 @@
 
 /* CSS related to settings page realm icon upload widget only */
 #realm-icon-upload-widget {
-    width: 100px;
-    height: 100px;
     border-radius: 5px;
-    box-shadow: 0 0 10px hsla(0, 0%, 0%, 0.1);
+    width: 100px;
 
     .image-delete-text {
         top: 40px;
         right: 18px;
     }
 
+    .image-upload-background {
+        width: 100px;
+        height: 100px;
+        box-shadow: 0 0 10px hsla(0, 0%, 0%, 0.1);
```

Comment:
@ArunSankarKs I think there is no need for box-shadow in image-upload-background.

Reply:
yeah, I agree. this doesn't effect the styling